From 34469063bb13439118ba367cf375407cdc0fad04 Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@cloudera.com>
Date: Thu, 25 Feb 2016 15:01:08 -0800
Subject: [PATCH 1275/1280] CLOUDERA-BUILD. Revert "svn merge -c 1580977 from
 trunk for HADOOP-10015. UserGroupInformation
 prints out excessive warnings."

This reverts commit 724e0ee4456bfe3986a057bf0b860a83d5b72381.
See CDH-37700

Change-Id: 724e0ee4456bfe3986a057bf0b860a83d5b72381f
(cherry picked from commit e578bcce12d112a712f1f86987bf2ec7ab27bed8)
---
 .../hadoop/security/UserGroupInformation.java      |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
index 86ab1d6..c7c52e8 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
@@ -1693,9 +1693,7 @@ protected Subject getSubject() {
       return Subject.doAs(subject, action);
     } catch (PrivilegedActionException pae) {
       Throwable cause = pae.getCause();
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("PrivilegedActionException as:" + this + " cause:" + cause);
-      }
+      LOG.warn("PriviledgedActionException as:"+this+" cause:"+cause);
       if (cause instanceof IOException) {
         throw (IOException) cause;
       } else if (cause instanceof Error) {
-- 
1.7.9.5

